gateway/src/apicast/policy/find_service/find_service.lua

```diff
@@ -70,8 +70,11 @@ function _M.new(...)
   return self
 end
 
-function _M:rewrite(context)
-  context.service = self.find_service(context.configuration, context.host)
+local function find_service(policy, context)
+  context.service = policy.find_service(context.configuration, context.host)
 end
 
+_M.rewrite = find_service
+_M.ssl_certificate = find_service
```

Comment:
why?

Reply:
ssl_certificate executes first and needs to have access to the service so it can execute the chain

I'm going to verify the context is shared or not after removing https://github.com/3scale/apicast/pull/622#discussion_r193339476. But then it still needs to find the service in both (but can skip if the service is already found).